test-data/unit/deps.test

```diff
@@ -1315,3 +1315,35 @@ def h() -> None:
 <m.D.__init__> -> m.h
 <m.D.__new__> -> m.h
 <m.D> -> m.D, m.h
+
+[case testLogicalSuperPython2]
+# flags: --logical-deps --py2
+class A:
+    def __init__(self):
+        pass
+    def m(self):
+        pass
+class B(A):
+    def m(self):
+        pass
+class C(B):
+    pass
+class D(C):
+    def __init__(self): # type: () -> None
+        super(B, self).__init__()
+    def mm(self): # type: () -> None
```

Comment:
Why function signature on the same line here and above? This looks unusual.

Reply:
No particular reason. I'll move them to separate lines.